courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<p>I'd remove the comments from the code since your code is already self-documented here. I'd put a paragraph in markdown if you think something more needs to be explained. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>